internal/store/store.go

```diff
@@ -529,7 +534,11 @@ func (bs *BlockStore) saveBlockToBatch(batch dbm.Batch, block *types.Block, bloc
 	height := block.Height
 	hash := block.Hash()
 
-	if g, w := height, bs.Height()+1; bs.Base() > 0 && g != w {
+	block_base, err := bs.Base()
+	if err != nil {
+		return pkg_errors.Wrap(err, "Error in getting block base")
+	}
```

Comment:
It's probably not necessary to even check that the `bs.Base() > 0` because an empty block store will return a height of 0 and we can check that instead seeing as the data is already there